sdk/keyvault/azure-keyvault-secrets/samples/hello_world.py

```diff
@@ -30,11 +30,13 @@
 #
 # ----------------------------------------------------------------------------------------------------------
 
+# [START create_secret_client]
 # Instantiate a secret client that will be used to call the service.
 # Here we use the DefaultAzureCredential, but any azure-identity credential can be used.
 VAULT_URL = os.environ["VAULT_URL"]
```

Comment:
Again, I think we can leave out the comments since the README gives context:
```suggestion
# Instantiate a secret client that will be used to call the service.
# Here we use the DefaultAzureCredential, but any azure-identity credential can be used.
# [START create_secret_client]
VAULT_URL = os.environ["VAULT_URL"]
```